.travis.yml

```diff
@@ -1,5 +1,11 @@
 language: node_js
-node_js: "6"
+os:
+  - linux
+  - osx
+node_js:
+  - "node"
```

Comment:
I guess that the only meaningful version is the one that is going to be used when publishing the extension. I wouldn't consider <6 either.